static/css/backtoTop.less

```diff
@@ -0,0 +1,26 @@
+#back2Top {
+    width: 40px;
+    line-height: 40px;
+    overflow: hidden;
+    z-index: 999;
+    display: none;
+    cursor: pointer;
+    -moz-transform: rotate(270deg);
+    -webkit-transform: rotate(270deg);
+    position: fixed;
+    bottom: 50px;
+    right: 0;
+    background-color: @lighter-grey;
+    text-align: center;
+    font-size: 30px;
+    text-decoration: none;
+}
+
+#back2Top:hover {
```

Comment:
LESS allows you to nest this in the CSS block above

e.g.
`
#back2Top {
  &:hover {
     ...
  }
}
`

This will help us keep these files maintainable.